### DISCUSSION TITLE
Can Directus support other schemas for MS SQL?

### DISCUSSION THREAD

**Discussion by calsch03** [directus/directus#4908] [Archive]
Hello, I'm trying to implement Directus to an existing Microsoft SQL database server. I connected Directus with the database, but no collections are visible in Directus. I use multiple schemas in my current database. Can Directus work with multiple schemas or can it only use the default dbo schema?

Database example:
![image](https://user-images.githubusercontent.com/19907926/114048804-2d21d480-988b-11eb-8ac8-a2fb90be0f99.png)

**Comment by rijkvanzanten** [comment]
Heya! There's currently no way to set the schema used for MS SQL unfortunately. Postgres' connection through `pg` allows you to configure the search path on a global level, but `tedious` (the connector for MS SQL) has no such thing. 

We're looking into implementing a way to dynamically add the schema to every query made within Directus, but that's a little tricky cross-database, so that's TBD

**Reply by calsch03** [reply]
Tnx for your quick response. Directus looks to be a good fit for our problems. It might be possible to migrate the database to PostgreSQL in the future.

**Reply by aguldbrandsen** [reply]
Hi! Are there still no options to set another schema than the dbo schema on MS SQL? I have tried using a SQL user that has default schema set to directus, and the tables are created when deploying but the application doesnt start succesfully.  

![image](https://github.com/directus/directus/assets/43814895/bc029ae4-1c0f-4fd1-b5e3-06339486af1f)

**Comment by edamim** [comment]
@rijkvanzanten 
Hi, just checking in — has there been any progress or official solution for using a custom MSSQL schema with Directus? We're trying to integrate with an existing Azure MSSQL database that uses multiple schemas, but Directus only seems to work with dbo, which is currently a blocker for us.
